add an faq about async commit or 1PC not working (#5494)
pingcap/docs: faq/deploy-and-maintain-faq.md

- The two results are calculated in different ways. `information_schema.tables.data_length` is an estimated value by calculating the averaged length of each row, while the store size on the TiKV monitoring panel sums up the length of the data files (the SST files of RocksDB) in a single TiKV instance.
- `information_schema.tables.data_length` is a logical value, while the store size is a physical value. The redundant data generated by multiple versions of the transaction is not included in the logical value, while the redundant data is compressed by TiKV in the physical value.

> #### Why does the transaction not have the Async Commit enabled or become a one-phase commit?

Reviewer: ```suggestion
#### Why does the transaction not use the Async Commit or the one-phase commit feature?
```

Reviewer 2: lgtm

> In the following conditions, even you enable [Async Commit](/system-variables.md#tidb_enable_async_commit-new-in-v50) using the system variables and [one-phase commit](/system-variables.md#tidb_enable_1pc-new-in-v50), TiDB will not use the features:

Reviewer: ```suggestion
In the following situations, even you have enabled the [Async Commit](/system-variables.md#tidb_enable_async_commit-new-in-v50) feature and the [one-phase commit](/system-variables.md#tidb_enable_1pc-new-in-v50) feature using the system variables, TiDB will not use these features:
```

Reviewer 2: lgtm

> - If you enable TiDB Binlog, limited by the implementation principles of TiDB Binlog, TiDB will not have Async Commit or one-phase commit enabled to use the features.

Reviewer: ```suggestion
- If you have enabled TiDB Binlog, TiDB, restricted by the implementation principles of TiDB Binlog, does not use the Async Commit or one-phase commit feature.
```

Reviewer 2: I cannot find the subject of the sentence "does not use the Async Commit or one-phase commit feature". And I think of course it is not an imperative sentence...

Reviewer: updated

Reviewer 2: And another bit. "Implementation principles" sounds a little weird to me. It's not something like a scientific law.
Maybe using"implementation mechanism" or just "limited by the implementation of .."?

Reviewer: "principles" are removed.

Reviewer 2: lgtm

> - Only when TiDB writes no more than 256 Key-value pairs in the transaction and the total size of keys is no more than 4KB, TiDB will have Async Commit or one-phase commit enabled to use the features. This is because for the transaction where the amount of data to be written is large, Async Commit is unable to greatly improve performance.

Reviewer: ```suggestion
- TiDB uses the Async Commit or one-phase commit features only when no more than 256 key-value pairs are written in the transaction and the total size of keys is no more than 4 KB. This is because, for transactions with a large amount of data to write, using Async Commit cannot greatly improve the performance.
```

Reviewer 2: lgtm.

> - Only when TiDB writes no more than 256 Key-value pairs in the transaction and the total size of keys is no more than 4KB, TiDB will have Async Commit or one-phase commit enabled to use the features. This is because for the transaction where the amount of data to be written is large, Async Commit is unable to greatly improve performance.

Reviewer: ```suggestion
- TiDB uses the Async Commit or one-phase commit features only when no more than 256 key-value pairs are written in the transaction and the total size of keys is no more than 4 KB. This is because, for transactions with a large amount of data to write, using Async Commit cannot greatly improve the performance.
```
